pkgs/by-name/en/energyplus/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  python3,
+}:
+stdenv.mkDerivation rec {
+  pname = "energyplus";
+  version = "24.1.0";
+
+  src = fetchFromGitHub {
+    owner = "NREL";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-g8lWijcZvNkFclSiStU+7HWfm+F8LobP2kIJNV6zczE=";
+  };
+
+  buildInputs = [
+    cmake
+    python3
+  ];
+
+  installPhase = ''
+    mkdir -p $out
+    export SRC=$(pwd)
+
+    cd $out
+    cmake $SRC
+
+    cd $SRC
+    make -j 32
```

Comment:
This is unusual — what's all this doing in installPhase?

Reply:
I'm not sure anymore either. I'm not that skilled with packaging yet and I was just trying to get it to run at some point and this helped I guess. But it seems unnecessary now indeed. I'll clean it up :+1: